integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -643,6 +637,13 @@ private int releaseInternal(String path, long fd) {
     } catch (Throwable e) {
       LOG.error("Failed to release {}", path, e);
       return -ErrorCodes.EIO();
+    } finally {
+      if (is != null) {
```

Comment:
curios, why is is not null, then remove fd?

Reply:
release can release input stream or output stream
expected only one stream is not null, that stream should be released and removed